### ISSUE TITLE
[module:download] choose a version for ...

### ISSUE THREAD

**Issue by novia713** [hechoendrupal/DrupalConsole#2845]
when downloading a module with composer (deprecated, but i still love it) it shows
`Please, choose a Packagist version`
this have no sense, because besides it's harcoded in code to grab the module from https://packagist.drupal-composer.org it is not relevant and the user would use another repo
(there was an option for that in  `config.yml`)

![seleccion_015](https://cloud.githubusercontent.com/assets/423279/19627710/ab3b8984-994d-11e6-9302-48bc8da631f8.png)

So, in short, as user i would prefer a message mentioning the module name, like
```
Please, select a version for "devel":
  [0] dev-8.x-3.x
  [1] 8.3.0
```

**Comment by novia713** [comment]
i'm working on this, @jmolivas  if you don't see inappropiate

**Comment by novia713** [comment]
also, another bug

Composer's infos (about using `composer require` instead and about `shellexec_output`) are shown ALSO when we are **not** using `--composer` option

**Comment by novia713** [comment]
![seleccion_016](https://cloud.githubusercontent.com/assets/423279/19628820/a951ee88-9966-11e6-8d08-bb1c9348355b.png)

PR at #2846

**Comment by novia713** [comment]
@jmolivas i don't see where these infos are fired up at `module:download` execution :disappointed: 

![seleccion_017](https://cloud.githubusercontent.com/assets/423279/19628827/e38e3480-9966-11e6-9fcf-d08efa65ca42.png)

**Comment by jmolivas** [comment]
@novia713 not sure I need to double-check where are those messages shown.

**Comment by novia713** [comment]
hi @jmolivas any idea on where to search for these messages?

**Comment by novia713** [comment]
* deleted tip about «shell_exec» at the following PRs:
https://github.com/hechoendrupal/drupal-console-en/pull/37
https://github.com/hechoendrupal/drupal-console-es/pull/10

 * the tip about using composer instead of `--composer` i'm unable to find where is it
